snap/local/config/gzdoom.ini

```diff
@@ -0,0 +1,88 @@
+[GlobalSettings]
+cl_run=true
+defaultiwad=Freedoom: Phase 1
+freelook=false
+m_forward=0
+snd_musicvolume=1
+gl_texture_filter=0
+
+[Doom.ConsoleVariables]
+am_overlay=2
+am_portaloverlay=true
+am_rotate=1
+
+[Doom.Bindings]
+1=slot 1
+2=slot 2
+3=slot 3
+4=slot 4
+5=slot 5
+6=slot 6
+7=slot 7
+8=slot 8
+9=slot 9
+0=slot 0
+-=sizedown
+Equals=sizeup
+Tab=togglemap
+W=+forward
+E=+use
+T=messagemode
+LeftBracket=invprev
+RightBracket=invnext
+Enter=invuse
+Ctrl=+attack
+A=+moveleft
+S=+back
+D=+moveright
+`=toggleconsole
+Shift=+speed
+\=+showscores
+,=+moveleft
+.=+moveright
+Alt=+strafe
+Space=+use
+CapsLock=toggle cl_run
+F1=menu_help
+F2=menu_save
+F3=menu_load
+F4=menu_options
+F5=menu_display
+F6=quicksave
+F7=menu_endgame
+F8=togglemessages
+F9=quickload
+F10=menu_quit
+F11=bumpgamma
+F12=spynext
+SysRq=screenshot
+Pause=pause
+Home=land
+UpArrow=+forward
+LeftArrow=+left
+RightArrow=+right
+DownArrow=+back
+Ins=+movedown
+Mouse1=+attack
+Mouse2=+strafe
+Mouse3=+forward
+Mouse4=+speed
+Joy1=+attack
+Joy2=+strafe
+Joy3=+speed
+Joy4=+use
+MWheelUp=weapprev
+MWheelDown=weapnext
+MWheelRight=invnext
+MWheelLeft=invprev
+DPadUp=togglemap
+DPadDown=invuse
+DPadLeft=invprev
+DPadRight=invnext
+Pad_Start=pause
+Pad_Back=menu_main
+LShoulder=weapprev
+RShoulder=weapnext
+LTrigger=+altattack
+RTrigger=+attack
+Pad_A=+use
```

Comment:
There are a lot of keys in this file that seem like they are better left out, keeping only the ones changed from GZDoom's default. I assume the port can handle a partial config file?

Reply:
Unfortunately, it doesn't work like that with binds; if you provide a binds section and you don't have stuff bound, it will not use any of the defaults

Reply:
That's too bad. Sounds like a bug to me, but oh well.